### PR TITLE
version 2.0

### PR DESCRIPTION
**Pull request by jakevdp**

### CODE REVIEW

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
-{% set version = "1.2.1" %}
+{% set version = "2.0.1" %}
 
 package:
   name: altair
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/a/altair/altair-{{ version }}.tar.gz
-  sha256: c1303f77f1ba4d632f2958c83c0f457b2b969860b1ac9adfb872aefa1780baa7
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @jakevdp I switched back to PyPI b/c the checksum is stable, GitHub changes it if there is an change in the version tag.

**Comment by jakevdp** [comment]
Cool, thanks for looking into this! Is it ready to merge?

**Comment by ocefpaf** [comment]
> Cool, thanks for looking into this! Is it ready to merge?

If you think this is ready merge away :smile: 

(Sorry for all those commits :flushed: the web-interface is handy but entourages messy commits. We can always squash and merge though.)

**Comment by jakevdp** [comment]
Looks good on my end

**Pull request merged**